haechan45th/src/controllers/postController.js

```diff
@@ -0,0 +1,119 @@
+const postService = require('../services/postService');
+
+const getAllPost = async (req, res) => {
+    try {
+        const posts = await postService.getAllPosts();
+        res.status(200).json(posts);
+    } catch (err) {
+        console.log(err);
+        res.status(400).json({ message: "Error getting All posts /postController" });
+    }
+};
+
+const getSpecificUserPost = async (req, res) => {
+    try {
+        const userId = req.params.userId;
+        if (!userId) {
+            return res.status(400).json({ message: 'USER_POSTS_NOT_FOUND' });
+        }
+
+        const posts = await postService.getSpecificUserPost(userId);
+
+        return res.status(200).json(posts);
+    } catch (err) {
+        console.log(err);
+        return res.status(400).json({ message: "Error getting Users posts /postController" });
+    }
+};
+
+const createUserPost = async (req, res) => {
+    try {
+        const userId = req.params.userId;
+        const { title, content, imageUrl } = req.body;
+
+        if (!userId || !title || !content) {
+            return res.status(400).json({ message: 'USER_POSTS_CANT_CREATED' });
+        }
+
+        await postService.createPosts(userId, title, content, imageUrl, res);
+
+        return res.status(202).json({
+            message: 'SUCCESSFULLY CREATED USER POST'
+        });
+    } catch (err) {
+        console.log(err);
+        return res.status(err.statusCode || 500).json({
+            message: err.message
+        });
+    }
+};
+
+const editUserPosts = async (req, res) => {
+    try {
+        const userId = req.params.userId;
+        const postId = req.params.postId;
```

Comment:
```suggestion
        const { userId, postId } = req.params;
```
구조분해할당 활용하면 코드를 가독성있고 line을 줄일 수 있습니다 :)